Gemfile.lock

```diff
@@ -1,58 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.3)
-      actionview (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-      rack (~> 2.0, >= 2.0.8)
+    actionpack (6.1.3)
+      actionview (= 6.1.3)
+      activesupport (= 6.1.3)
+      rack (~> 2.0, >= 2.0.9)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.3)
-      activesupport (= 6.0.3.3)
+    actionview (6.1.3)
+      activesupport (= 6.1.3)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activemodel (6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activerecord (6.0.3.3)
-      activemodel (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
-    activesupport (6.0.3.3)
+    activemodel (6.1.3)
+      activesupport (= 6.1.3)
+    activerecord (6.1.3)
+      activemodel (= 6.1.3)
+      activesupport (= 6.1.3)
+    activesupport (6.1.3)
       concurrent-ruby (~> 1.0, >= 1.0.2)
-      i18n (>= 0.7, < 2)
-      minitest (~> 5.1)
-      tzinfo (~> 1.1)
-      zeitwerk (~> 2.2, >= 2.2.2)
+      i18n (>= 1.6, < 2)
+      minitest (>= 5.1)
+      tzinfo (~> 2.0)
+      zeitwerk (~> 2.3)
     builder (3.2.4)
-    concurrent-ruby (1.1.7)
+    concurrent-ruby (1.1.8)
     crass (1.0.6)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.9.0)
-    i18n (1.8.5)
+    erubi (1.10.0)
+    i18n (1.8.9)
       concurrent-ruby (~> 1.0)
-    json (2.3.1)
+    json (2.5.1)
     jwt (2.2.2)
-    jwt_signed_request (2.6.0)
+    jwt_signed_request (3.0.0)
       jwt (>= 1.5.0)
       rack
-    loofah (2.7.0)
+    loofah (2.9.0)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     method_source (1.0.0)
-    mini_portile2 (2.4.0)
-    minitest (5.14.2)
-    money (6.13.8)
+    mini_portile2 (2.5.0)
+    minitest (5.14.4)
```

Comment:
### [minitest](https://github.com/seattlerb/minitest)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 5.14.2 → 5.14.4

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **8** commits. See the full changes on [the compare page](https://github.com/seattlerb/minitest/compare/v5.14.2...v5.14.4).

These are the individual commits:
- (8e55b9b) [- Bumped require_ruby_version to < 4 (trunk = 3.1).](https://github.com/seattlerb/minitest/commit/8e55b9b07b31efe6734a7aff81df0535c15000c2)
- (b8fb101) [prepped for release](https://github.com/seattlerb/minitest/commit/b8fb1014ad1932eddefb5107c0f6504ba45e0cbc)
- (94faf7b) [Added a support matrix to the FAQ in README.rdoc](https://github.com/seattlerb/minitest/commit/94faf7b90cbe03b49d34d43f17cc0ea06eb8c53f)
- (9eb312b) [- Fixed deprecation warning using stub with methods using ke…](https://github.com/seattlerb/minitest/commit/9eb312bea1276117fbacb09a1c8bb7abd1434397)
- (dfbde40) [- ! Unfixed deprecation warning fix that broke in nearly eve…](https://github.com/seattlerb/minitest/commit/dfbde407800d3a5dbb54ec9461635f062c2fe272)
- (8a7ba81) [- Re-fixed deprecation warning using stub with methods using…](https://github.com/seattlerb/minitest/commit/8a7ba8191f6d09d5e7dc04d53054e8b194e61fe4)
- (13c48a0) [Improved test_stub_callable_keyword_args to test pass-throug…](https://github.com/seattlerb/minitest/commit/13c48a03d84a2a87855a4de0c959f96800100357)
- (3c6576a) [prepped for release](https://github.com/seattlerb/minitest/commit/3c6576a51f4e266996e3459d7a0dd054eb4c87f7)


</details>